doc/manual-wiki/misc.wiki

```diff
@@ -28,9 +28,8 @@ The function <<a_api | value Eliom_content.Html.D.make_uri>>
 
 
 
-To simplify the creation of <<span class="code"|<link>>> tags
-      for CSS or <<span class="code"|<script>>> tags for Javascript,
-        use the following functions:
+To simplify the creation of {{{<link}}}> tags for CSS or {{{<script}}}>
```

Comment:
Be careful with nesting :). `>` should go inside `{{{ }}}`.

Reply:
Ah, good catch! Thanks :)